ui/plugins/pinboard.js

```diff
@@ -126,6 +126,12 @@ treeherder.controller('PinboardCtrl', [
         };
 
         $scope.canSaveClassifications = function() {
+            if ($scope.enteringBugNumber) {
+                // we should save this for the user, as they likely
+                // just forgot to hit enter.
+                $scope.saveEnteredBugNumber();
+            }
+            $scope.completeClassification();
```

Comment:
I'm not totally clear on why ``$scope.completeClassification()`` is needed here.  This gets called in the ``$scope.save()`` function, right?  If I'm missing something, then please feel free to leave it.  :)